src/common/cache/tests/it/main.rs

```diff
@@ -12,4 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#![allow(clippy::uninlined_format_args)]
```

Comment:
I've tried tweaking Clippy by adding a `clippy.toml` file, but this option not supported.

This can be done by making `Clippy` running with a `-Auninlined_format_args` passed in CLI, but I'm afraid this may lead to more and more tailing `-A`s.

But on the other hand, adding allows for every crate leaves them the freedom of whether to enable it in the future.